core/src/test/java/org/elasticsearch/deps/joda/SimpleJodaTests.java

```diff
@@ -314,6 +314,28 @@ public void testForInvalidDatesInEpochMillis() {
         }
     }
 
+    public void testForInvalidTimeZoneWithEpochSeconds() {
+        DateTimeZone zone = DateTimeZone.forOffsetHours(1);
+        FormatDateTimeFormatter formatter = new FormatDateTimeFormatter("epoch_seconds", new DateTimeFormatterBuilder().append(new Joda.EpochTimeParser(false)).toFormatter().withZone(zone), Locale.ROOT);
```

Comment:
The length of this line will trigger our checkstyle rules once we run the tests, we were at 140 characters (still applies to this test class) and recently moved to 100. Can you shorten this before I kick of the first CI run?